models/netsuite2/src_netsuite2.yml

```
@@ -286,6 +290,41 @@ sources:
           - name: _fivetran_deleted
             description: "{{ doc('_fivetran_deleted') }}"
 
+      - name: employee
+        identifier: "{{ var('netsuite2_employee_identifier', 'employee') }}"
+        description: "{{ doc('employee_table') }}"
+        columns:
```

Apologies I didn't catch this after the addition of the `netsuite2__using_employees` variable, but we need to add a config here to allow the disablement of the employee source if it isn't being used.

```suggestion
        description: "{{ doc('employee_table') }}"
        config:
          enabled: "{{ var('netsuite2__using_employees', true) }}"
        columns:
```